<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=\, initial-scale=1.0">
    <title>ProtVision</title>

    <link rel="stylesheet" href="style.css">
</head>
<body>
    
    <header class="header">
        <h1>ProtVision</h1>
        <p id="author" >Developed by Rohan Raj Samal, <a href="https://divyaranjan16.github.io/divya-portfolio/" target="_blank">Divya Ranjan Pradhan</a> and Shailesh Kumar </p>        
    </header>


    <!-- Tools section with description -->
    <section id="tools-area" >
        <h3 id="Introduction" >All the tools listed below can be used to get insights into the protein sequence of interest by giving their respective UniProt Accession ID, by entering the sequence directly or through file input. <br>
        But we are not limited to only proteins and are continuously working to make tools for other analysis as well. <br>
</h3>

        <h4>We recommend to use the tools in your Computer systems and not in mobile devices. <br>
            *Click on the name to use the tool.</h4>
            <ul id="tools-list" >
                 <li><p><a href="./aliphatic_index_web/index.html" target="_blank" >Aliphatic Index</a>: 
                    
                    This tool will give you the Aliphatic Index of the protein sequence. <br>
                    The aliphatic index of a protein is a measure of the relative volume occupied by aliphatic side chains of the amino acids alanine, valine, leucine and isoleucine. It is calculated as the sum of the relative volumes of these amino acids divided by the total number of amino acids in the protein. An increase in the aliphatic index increases the thermostability of globular proteins</p></li>


                 <li><p><a href="./amiino_acid_comp/index.html" target="_blank" >Amino Acid Composition</a>: 
                    
                    This tool will give you the percentage of each amino acid residue present in the protein sequence. 
                    The value obtained can help us understand the nature of the protein.</p></li>


                <li><p><a href="./extinction/index.html"  target="_blank">Extinction Coefficient</a>: 
                    
                    This tool will give you the Extinction Coefficient and the Absorbance of the protein sequence. <br>
                    </p></li>


                 <li><p><a href="./GRAVY/index.html"  target="_blank">GRAVY</a>:
                    
                    This tool will give you the Grand Average of Hydropathy (GRAVY) value of the protein sequence. <br>
                    The GRAVY value of a protein is a measure of its hydrophobicity or hydrophilicity. It is calculated by adding the hydropathy values of each amino acid residues and dividing by the number of residues in the sequence or length of the sequence. Increasing positive score indicates a greater hydrophobicity. Negative GRAVY value indicates that the protein is non-polar and positive value indicates that the protein is polar</p></li>


                 <li><p><a href="./half_life/index.html"  target="_blank">Half Life</a>: 
                    
                    This tool will give you the Half Life of the protein sequence. <br>
                    The half-life of a protein is the time after which the amount of the protein decreases by 50%. If a protein has a half-life of 1 hour, then 1 hour later, you'll have 50% of the original amount of the protein left. The average half-life of a protein is within the range of 4-14 hours, with the median half-life being 8.7 hours. The majority of proteins have half-lives less than 8 hours</p></li>


                 <!-- <li><p><a href="./instability index/index.html"  target="_blank">Instability Index</a>: 
                    
                    This tool will give you the Instability Index of the protein sequence. <br>
                    The instability index is a measure of proteins that predicts the stability of a protein based on its amino acid composition. If the instability index is less than 40, then the protein is predicted to be stable in a test tube. If the instability index is greater than 40, then the protein is predicted to be unstable.</p></li> -->


                 <li><p><a href="./molecular_weight/index.html"  target="_blank">Molecular Weight</a>:
                    
                    This tool will give you the total Molecular Weight of the protein sequence.</p></li>


                 <li><p><a href="./PI/index.html"  target="_blank">Isoelectric Point (pI)</a>:
                    
                    This tool will give you the Isoelectric Point of the protein sequence. <br>
                    The isoelectric point (pI) is the point at which the net charge on a molecule is zero. pI is most commonly examined for proteins. Each of the amino acids in a protein carries a distinct charge, and the overall charge of a protein is the summation of the individual charges on each amino acid. This charge is also dependent on the pH of the surrounding solution.</p></li>
            </ul>
    </section>
    <div class="foot" >
        <p>All these tools are free to use. Use them to the fullest.</p>
        <br>
        <p>Got Queries? Contact Us at siliconbiology@gmail.com  or genomeinquisitors@gmail.com</p>
    </div> 
</body>
</html>
